### PR TITLE
fix: #1323 delete zk registry when set defualt consumer/provider config

### LINKED ISSUE #1323: 关于默认配置依赖zk的问题
当用户没有提供有效配置时，目前默认配置会加上对zk注册中心的配置，这样会导致即使用户什么都没做，他也需要启动一个zk环境，我们这并不是一个合理的行为。
因此，我们考虑默认配置中应该下掉对zk注册中心的依赖。
image

### PR DESCRIPTION
**Pull request by Mulavar**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #1323 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: config/consumer_config.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func SetConsumerConfig(c ConsumerConfig) {
 // ConsumerInit loads config file to init consumer config
 func ConsumerInit(confConFile string) error {
 	if confConFile == "" {
-		return perrors.Errorf("application configure(consumer) file name is nil")
```
Comment (zouyx) [inline config/consumer_config.go:86]: why delete this error？

File: config/provider_config.go [hunk 2]
```diff
@@ -70,7 +70,7 @@ func SetProviderConfig(p ProviderConfig) {
 // ProviderInit loads config file to init provider config
 func ProviderInit(confProFile string) error {
 	if len(confProFile) == 0 {
-		return perrors.Errorf("application configure(provider) file name is nil")
```
Comment (zouyx) [inline config/provider_config.go:73]: as above

File: config/consumer_config.go (hunk 1 above)
Comment (Mulavar) [reply config/consumer_config.go:86]: 这部分需要结合config_loader.go来看，初始化provider和consumer config有这么一段逻辑
if errCon := ConsumerInit(confConFile); errCon != nil {
    consumerConfig = nil
} else if confBaseFile == "" {
    setDefaultValue(consumerConfig)
}
为了保证file==""的时候能走到setdefaultvalue的分支，因此这里改成return nil

File: config/provider_config.go (hunk 2 above)
Comment (Mulavar) [reply config/provider_config.go:73]: as above

**Comment by LaurenceLiZhixin** [comment]
Pls fix your conflict and just push me to revew, tks.

**Review by cjphaha — APPROVED** [review]

**Comment by cjphaha** [comment]
LGTM

**Review by xavier-niu — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**